database.go

```diff
@@ -102,7 +128,10 @@ func (f *FirestoreRecurserDB) GetByUserID(ctx context.Context, userID, userEmail
 		recurser := doc.Data()
 		recurser["name"] = userName
 		recurser["email"] = userEmail
-		r = MapToStruct(recurser)
+		r, err = MapToStruct(recurser)
+		if err != nil {
+			return Recurser{}, fmt.Errorf("getting database entry for recurser %s (ID: %d)", userName, userID)
```

Comment:
Assuming this is a dev-focused error message, I'd want to see the causing error too, since it's easy to get:
```suggestion
			return Recurser{}, fmt.Errorf("getting database entry for recurser %s (ID: %d): %w", userName, userID, err)
```

Reply:
Yes, absolutely! I'd meant to do this, missed this one.